hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rollout/ProxyRollout.java

```diff
@@ -18,113 +19,122 @@
  */
 public class ProxyRollout extends Rollout {
```

Comment:
Javadoc is already provided, corrected typo mistake.
Please let me know that expected is same?.